### DISCUSSION TITLE
fpm on Windows

### DISCUSSION THREAD

**Discussion by Ily83** [fortran-lang/fpm#571] [General]
Is fpm not working through MSYS2 on Windows for anyone else?

![image](https://user-images.githubusercontent.com/90933947/133847371-19462391-ba03-4567-b3f1-df9883df4d85.png)

**Comment by awvwgk** [comment]
I would need a little more information to help here. I packaged fpm for mingw64, mingw32 and ucrt64 runtimes, your terminal seems to be a msys runtime. Not sure if this is supposed to have access to the native MinGW binaries, but you might just try one of the three terminals mentioned above (probably mingw64).

**Comment by Ily83** [comment]
You're right, it works. Thank you.
